packages/core/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@usedapp/core",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "repository": "git@github.com:EthWorks/dapp-framework.git",
+  "author": "Justyna <jusbroniszewska@gmail.com>",
```

```suggestion
  "author": "Ethworks",
```

@JustynaBroniszewska If you're OK with this :D

---

hah, of course 😄